### DISCUSSION TITLE
sidekiq-pro with dragondlfydb

### DISCUSSION THREAD

**Discussion by sundar-dt** [dragonflydb/dragonfly#4066] [General]
When I use Sidekiq-pro 7.3.2 with dragonflyd 1.23 I am getting below error, how to solve this?

W20241104 19:56:11.438251    13 main_service.cc:1459]  EVALSHA f8133be7f04abd9dfefa83c3b29a9d837cfbda86 2 retry queues 1730750171.385535 failed with reason: -NOSCRIPT No matching script. Please use EVAL.
W20241105 05:55:08.606178    16 script_mgr.cc:313] Error running script (call to 34b1048274c8e50a0cc587a3ed9c383a82bb78c5): @user_script:11: script tried accessing undeclared key, key: orphan-67a584d47c87054c1c72074d
W20241105 05:55:08.606292    16 main_service.cc:1459]  EVALSHA 34b1048274c8e50a0cc587a3ed9c383a82bb78c5 3 queue:sq|sidekiq-notification-dev-deployment-6bf645f9-ksjml:8:1795a85e969f|PRODUCTION_critical_notifications queue:PRODUCTION_critical_notifications dead 3 259200 1730786108.5451775 failed with reason: Error running script (call to 34b1048274c8e50a0cc587a3ed9c383a82bb78c5): @user_script:11: script tried accessing undeclared key, key: orphan-67a584d47c87054c1c72074d

**Comment by kostasrim** [comment]
Hello @sundar-dt , 

You have to configure dragonfly properly. Plz refer to https://www.dragonflydb.io/docs/integrations/sidekiq
